terraform/accounts/main/jenkins_instance_profile.tf

```diff
@@ -143,3 +143,25 @@ resource "aws_iam_role_policy" "jenkins" {
 }
 EOF
 }
+
+data "aws_iam_policy_document" "cloudwatch-policy" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogStreams",
+    ]
+
+    resources = [
+      "arn:aws:logs:eu-west-1:*:*",
```

Comment:
Can we restrict this to our main account?

Reply:
Done